### PR TITLE
Minify CSS and JS

### PR DESCRIPTION
**Pull request by crisgon**
Minify CSS #1 
Minify JS #2

### CODE REVIEW

**Comment by vini23hot** [comment]
Thank you for your contribution! However, **it's needed to minify it and keep the main JS and CSS files** if we want to change them.

File: index.html [hunk 1]
```diff
@@ -35,7 +35,9 @@ <h2>Proxy simples e rápido para consulta de CEP</h2>
     </div>
   </header>
   <div class="main">
-    <section id="section-left">
+    <section id="section-left">thank you @VonderVuflya, but if we need to change some css of our own, what should we do?
```
Comment (vini23hot) [inline index.html]: I think you did a mistake adding this text here.

**Comment by crisgon** [comment]
Done

**Comment by rafa-acioly** [comment]
@crisgon we don't need to minify the `main.js` file, can you fix it?

**Pull request closed without merging**